draft-ietf-quic-recovery.md

```diff
@@ -522,9 +525,17 @@ removed from bytes in flight when the Initial and Handshake keys are discarded.
 ### Sending Probe Packets
 
 When a PTO timer expires, a sender MUST send at least one ack-eliciting packet
-as a probe, unless there is no data available to send.  An endpoint MAY send up
-to two full-sized datagrams containing ack-eliciting packets, to avoid an
-expensive consecutive PTO expiration due to a single lost datagram.
+in the packet number space as a probe, unless there is no data available to
+send.  An endpoint MAY send up to two full-sized datagrams containing
+ack-eliciting packets, to avoid an expensive consecutive PTO expiration due
+to a single lost datagram.
+
+In addition to sending data in the packet number space for which the timer
+expired, the sender SHOULD coalesce ack-eliciting packets from all other packet
+number spaces with in-flight data if sending coalesced packets is supported.
+If implementations do not send coalesced packets upon timeout when multiple
+packet number spaces have in-flight data, then they MUST only send a single
```

Comment:
Suggestion: "If an endpoint has data in multiple packet number spaces in flight and is unable to send coalesced packet when a PTO expires, it MUST send only a single datagram on PTO expiration."

Reply:
@janaiyengar has it right, but repeats "PTO expiration", suggest:

> If an endpoint has data in multiple packet number spaces in flight and is unable to send coalesced packet when a PTO expires, it MUST send only a single datagram.

Reply:
Done